src/main/java/uk/gov/hmcts/reform/wataskmonitor/schedulers/TaskConfiguratorScheduler.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.wataskmonitor.schedulers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.wataskmonitor.models.CamundaTask;
+import uk.gov.hmcts.reform.wataskmonitor.services.CamundaService;
+import uk.gov.hmcts.reform.wataskmonitor.services.TaskConfigurationService;
+
+import java.util.List;
+
+@Component
+@Slf4j
+@ConditionalOnProperty(
+    value = "task.configurator.scheduling.enable", havingValue = "true"
+)
+public class TaskConfiguratorScheduler {
```

Comment:
We probably want to switch this off as it could have unintended results if we were to deploy it now?  If each node is running on a different schedule and pick up the same TaskIds to process and attempt to run them concurrently.

Reply:
Yeah, it's off by default. To turn it on, we have to add that enable to true in the properties. I haven't removed it yet because we don't know exactly how we will deploy this app in Prod and it could become somehow useful. In any case, it's something subject to be cleaned up once we are clear about how to deploy this service.